packages/web-twig/Makefile

```diff
@@ -53,6 +54,9 @@ bash: ## Connect to the PHP container
 
 sh: bash ## Connect to the PHP container (alias for `bash`)
 
+cert: ## Trust the Authority of server certificates
```

Comment:
maybe we should mention, that this works only on macos?

Reply:
Oh, sorry, you are the one with the special needs OS ♿ Try it now ;-)

Reply:
Commands are based on https://github.com/dunglas/symfony-docker/blob/main/docs/tls.md

Reply:
Ok.